### ISSUE TITLE
Feature Request: Default CSS style for fieldset/legend elements

### ISSUE THREAD

**Issue by jumpinjackie** [palantir/blueprint#442]
If we wrap our beautiful blueprint-styled form elements with `<fieldset>` , it sticks out like a sore thumb.

Looking through the BP docs I don't see any CSS references for `<fieldset>` or `<legend>` elements, so I gather blueprint's CSS doesn't actually define anything for such elements.

So this is a request to give us a default fieldset/legend CSS style out of the box that we can use.

**Comment by llorca** [comment]
We use heading tags to add a heading to a form group. If we were to style fieldset/legend, it'd look like just a heading (`h4` or `h5`).

Do we want default styling for every HTML element? My hunch is no, but I'll let others chime in. https://github.com/palantir/blueprint/issues/244 may be related

**Comment by giladgray** [comment]
![image](https://cloud.githubusercontent.com/assets/464822/21819022/d4b5b24e-d71e-11e6-97b3-6d4364ee3993.png)

_whoa._ I was gonna say something about "we're trying not to style HTML elements directly see #244" but this is truly unpleasant.

> The HTML `<fieldset>` element is used to group several controls as well as labels (`<label>`) within a web form. [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset)

**Comment by llorca** [comment]
Yup, and it'd seem silly to create a `.pt-fieldset` class just to remove default styling. But also as you said, we're trying not to style HTML elements directly (#244). What's a good solution then? 😯

**Comment by giladgray** [comment]
well, simply adding `class="pt-card"` goes a long way towards making a `fieldset` attractive.

**Comment by adidahiya** [comment]
A `.pt-fieldset` class to remove default styling doesn't actually sound bad. I would prefer the behavior to be opt-in and as mentioned in #244, we should try to remove global styling rather than add it to Blueprint.

**Comment by jumpinjackie** [comment]
Just to clarify. By default styling I do mean something akin to a `.pt-fieldset` and not a global CSS rule for all `fieldset`/`legend` elements. 

@giladgray If applying `.pt-card` on a `fieldset` is sufficient as you would suggest, then some mention in the docs about this would suffice. Anything that a developer new to blueprint who adds a `fieldset` to group form elements does not have to ask the question: Why does my form look like $YOUR_ATTACHED_SCREENSHOT?
